flow/Arena.h

```diff
@@ -380,12 +380,6 @@ class Standalone : private Arena, public T {
 	}
 #else
 	Standalone( const T& t, const Arena& arena ) : Arena( arena ), T( t ) {}
-	Standalone( const Standalone<T> & t ) : Arena((Arena const&)t), T((T const&)t) {}
```

Comment:
Why are we removing these constructors? I think part of the design of Standalone is that copying a Standalone<T> should be cheap, see https://github.com/apple/foundationdb/tree/master/flow#arenas

> When a Standalone<T> is copied (e.g. by argument passing or assignment) to another Standalone<T>, they will share the same memory.

Reply:
Here we are just explicitly defining the default copy constructor, which disables the default move constructor. I removed this constructor in order to enable the default move constructor.

Reply:
Would
```
Standalone( const Standalone<T> & t ) = default;
Standalone( Standalone<T> && t ) = default;
...
```
work?

Reply:
I added these `=default` constructors for clarity.